### PR TITLE
Update more links

### PR DESCRIPTION
**Pull request by claughinghouse**
**Description:**

Updates the following links:
docs.micropython.org
Python requests
awesome-ha.com
layla.amazon.com
nestdevelopers.io
Tor Browser on Android

#10491 

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by fabaff — APPROVED** [review]

File: source/_posts/2018-07-21-release-74.markdown [hunk 1]
```diff
@@ -30,7 +30,7 @@ To try it out today, [read these instructions](https://developers.home-assistant
 
 We keep seeing [great examples](https://twitter.com/home_assistant/status/1019579208622845953) of UIs built with Lovelace. Follow us on social media ([FB](https://www.facebook.com/homeassistantio/?ref=bookmarks), [Twitter](https://twitter.com/home_assistant)) where we will keep sharing great examples.
 
-For the Lovelace changes in this release, check out the [changelog](/lovelace/changelog/). To help our development and design teams, we've also introduced a [Lovelace card gallery](https://home-assistant-lovelace-gallery.netlify.com/).
+For the Lovelace changes in this release, check out the [changelog](/lovelace/changelog/). To help our development and design teams, we've also introduced a [Lovelace card gallery](https://www.awesome-ha.com/).
```
Comment (frenck) [inline source/_posts/2018-07-21-release-74.markdown:33]: This is an incorrect change. The Awesome list is NOT the Lovelace card gallery.

Comment (frenck) [reply source/_posts/2018-07-21-release-74.markdown:33]: /CC @fabaff

Comment (claughinghouse) [reply source/_posts/2018-07-21-release-74.markdown:33]: @frenck apologies. How should I back out the changes? Create another PR with the revision back to the original?

Comment (frenck) [reply source/_posts/2018-07-21-release-74.markdown:33]: If you'd could create another PR that removed that part (the Gallery is not in use anymore), that would be awesome @claughinghouse 👍

Comment (claughinghouse) [reply source/_posts/2018-07-21-release-74.markdown:33]: Updated with PR #10696

Comment (fabaff) [reply source/_posts/2018-07-21-release-74.markdown:33]: No, it's not but it's better to send people to a place that exists than to a 404.

https://home-assistant-lovelace-gallery.netlify.com/ is no longer reachable.

Comment (frenck) [reply source/_posts/2018-07-21-release-74.markdown:33]: Agreed. My initial bet would be removing that link & text all together.

**Pull request merged**